commit 77d8b3c8262edd8e880399a1002ef6b1a0bbefcd
Author: Rafał Miłecki <rafal@milecki.pl>
Date:   Sat Jan 28 14:31:22 2017 +0100

    bcma: use (get|put)_device when probing/removing device driver
    
    commit a971df0b9d04674e325346c17de9a895425ca5e1 upstream.
    
    This allows tracking device state and e.g. makes devm work as expected.
    
    Signed-off-by: Rafał Miłecki <rafal@milecki.pl>
    Signed-off-by: Kalle Valo <kvalo@codeaurora.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1346_77d8b3c8262edd8e880399a1002ef6b1a0bbefcd.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1346_77d8b3c8262edd8e880399a1002ef6b1a0bbefcd.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
